Add 7.3.56, 7.3.57 release notes (#11795)
apple/foundationdb: documentation/sphinx/source/release-notes/release-notes-730.rst

Release Notes
#############

7.3.57
======
* Same as 7.3.56 release with AVX enabled.

7.3.56
======

> * Manual flush if the rocksdb flush does not happen within a time interval `(PR #11792) <https://github.com/apple/foundationdb/pull/11792>`_

Reviewer: ```suggestion
* Added an option to force flush if RocksCb flush does not happen within a time interval `(PR #11792) <https://github.com/apple/foundationdb/pull/11792>`_
```

> * Have RocksDB histogram sample rate updated to 1 `(PR #11794) <https://github.com/apple/foundationdb/pull/11794>`_

Reviewer: ```suggestion
* Changed RocksDB histogram sample rate to 1 `(PR #11794) <https://github.com/apple/foundationdb/pull/11794>`_
```

> * TLS should accept same key with different values. `(PR #11763) <https://github.com/apple/foundationdb/pull/11763>`_

Reviewer: @kakaiu what's the user impact for this PR? We need to state what problem this PR fixed here.

Reviewer 2: This PR solves a problem of using TLS. Sometimes, TLS input string can contain same NID with multiple different Criteria. The existing implementation uses std::map<NID, Criteria> with insert operation which can only store the left-most NID-Criteria pair in the TLS input string. This PR allows to use same NID with multiple different Criteria  as the user input string by using std::set<std::pair<NID, Criteria>> to store the Criteria.

> * Add knobs and metrics for bloom filter. `(PR #11785) <https://github.com/apple/foundationdb/pull/11785>`_

Reviewer: ```suggestion
* Added Sharded RocksDB knobs and metrics for bloom filters. `(PR #11785) <https://github.com/apple/foundationdb/pull/11785>`_
```

> * Fix backup dryrun bug caused by out of variable and lack of wait. `(PR #11774) <https://github.com/apple/foundationdb/pull/11774>`_

Reviewer: ```suggestion
* Fixed a backup agent crash bug when receiving an invalid token error from S3. `(PR #11774) <https://github.com/apple/foundationdb/pull/11774>`_
```

> * Add debug id trace event 7.3 in tagPartitionedLogSystem. `(PR #11776) <https://github.com/apple/foundationdb/pull/11776>`_

Reviewer: Drop this for no user/production impact.

> * Update ROCKSDB_MEMTABLE_MAX_RANGE_DELETIONS Knob to prevent OOMs `(PR #11739) <https://github.com/apple/foundationdb/pull/11739>`_

Reviewer: ```suggestion
* Updated ROCKSDB_MEMTABLE_MAX_RANGE_DELETIONS Knob to prevent OOMs of RocksDB storage servers `(PR #11739) <https://github.com/apple/foundationdb/pull/11739>`_
```

> * Enable backward read in consistency checker. `(PR #11743) <https://github.com/apple/foundationdb/pull/11743>`_

Reviewer: This is a simulation improvement, which can be dropped.
